contracts/dev-utils/src/wrappers.ts

```diff
@@ -4,5 +4,6 @@
  * -----------------------------------------------------------------------------
  */
 export * from '../generated-wrappers/dev_utils';
+export * from '../generated-wrappers/eth_balance_checker';
```

Comment:
Hm, this seems like a regression. We do _not_ want to export `EthBalanceChecker` from `contracts/dev-utils`. Please check that it's not in the `publicInterfaceContracts` key in `package.json` `configs`.

Reply:
Fixed in https://github.com/0xProject/0x-monorepo/pull/2321/commits/3fdadaf34a328e54737796d3f35e13d97c308a15